README.md

```diff
@@ -7,3 +7,24 @@ slog is a library for capturing structured log information. In contrast to "trad
 * captures arbitrary key-value metadata on each log event
 
 slog forwards messages to [`log`](https://golang.org/pkg/log/) by default. But you probably want to write a a custom output to make use of the context and metadata. At [Monzo](https://monzo.com/), slog captures events both on a per-service and a per-request basis (using the context information) and sends them to a centralised logging system. This lets us view all the logs for a given request across all the micro-services it touches.
+
+## Usage
+
+Internally at Monzo, we recommend that users always prefer structured logging where possible. An example of using slog for this would be:
+
+```go
+slog.Info(ctx, "Loading widget", map[string]interface{}{
+    "stage": "reticulating splines",
+})
+```
+
+### Other uses
+
+For backwards-compatibility, slog accepts metadata in the form of `map[string]string`.
+
+It also accepts format parameters in the style of `Printf`:
+
+```go
+stage := "reticulating splines"
```

Comment:
This brings back memories 💚 

![The Sims](https://usercontent1.hubstatic.com/14931662_f520.jpg)

Reply:
Oh wow, I didn't realize how retro that interface would look 20 years on.